test/functional/multi-database/multi-database-basic-functionality/multi-database-basic-functionality.ts

```diff
@@ -20,46 +20,29 @@ const VALID_NAME_REGEX = /^(?!sqlite_).{1,63}$/
 
 describe("multi-database > basic-functionality", () => {
     describe("filepathToName()", () => {
-        for (const platform of [`darwin`, `win32`]) {
-            let realPlatform: string
-
-            beforeEach(() => {
-                realPlatform = process.platform
-                Object.defineProperty(process, `platform`, {
-                    configurable: true,
-                    value: platform,
-                })
-            })
-
-            afterEach(() => {
-                Object.defineProperty(process, `platform`, {
-                    configurable: true,
-                    value: realPlatform,
-                })
-            })
-
-            it(`produces deterministic, unique, and valid table names for relative paths; leaves absolute paths unchanged (${platform})`, () => {
-                const testMap = [
-                    ["FILENAME.db", "filename.db"],
-                    ["..\\FILENAME.db", "../filename.db"],
-                    [
-                        "..\\longpathdir\\longpathdir\\longpathdir\\longpathdir\\longpathdir\\longpathdir\\longpathdir\\FILENAME.db",
-                        "../longpathdir/longpathdir/longpathdir/longpathdir/longpathdir/longpathdir/longpathdir/filename.db",
-                    ],
-                    ["C:\\dirFILENAME.db", "C:\\dirFILENAME.db"],
-                    ["/dir/filename.db", "/dir/filename.db"],
-                ]
-                for (const [winOs, otherOs] of testMap) {
-                    const winOsRes = filepathToName(winOs)
-                    const otherOsRes = filepathToName(otherOs)
-                    expect(winOsRes).to.equal(otherOsRes)
-                    expect(winOsRes).to.match(
-                        VALID_NAME_REGEX,
-                        `'${winOs}' is invalid table name`,
-                    )
-                }
-            })
-        }
+        if (process.platform !== "win32") return
```

Comment:
These tests shouldn't be skipped on non-windows platforms. On darwin/linux we still want to assert that relative paths are normalized when used as table names.